docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -1,240 +1,396 @@
+\begin{itemize}
+    \item Every \textbf{Monday} (excluding breaks \& holidays) from \textbf{2:30 PM to 4:20 PM} (2 hours) at the designated tutorial time. All team members are required to attend, be punctual, and come prepared with updates or questions.
+    
+    \item The meeting will be held in person in one of the following buildings: \textbf{MDCL, LRW, Gerald Hatch Center, or Mills Library}. Consult the team the weekend before the meeting to choose a suitable tutorial/meeting room and if necessary, reserve it for a 2 hour time period.
+    
+    \item \textbf{Contingency for in-person meetings}: If extenuating circumstances (e.g., severe weather) prevent meeting in person, the meeting will be held virtually or postponed to the next available date based on group consensus. Meetings may also be held virtually if discussed and agreed upon by the majority of the members. Virtual meetings will take place on our shared Discord server.
 
-\wss{How often will you meet with your industry advisor?  when?  where?}
+    \item When required, the team can host additional meetings throughout the week that align with each member’s schedules to address certain roadblocks/issues encountered throughout the working week. The meeting will take place in the same buildings listed above or virtual if there is no seating available.
+\end{itemize}
 
-\wss{Will meetings be virtual?  At least some meetings should likely be
-in-person.}
+\subsection{Meeting Structure}
+Each meeting will be scheduled in advance with an agenda prepared by the Meeting Secretary (more details on roles discussed in Section 4.3.). Attendance will be recorded at the start of each meeting, after which agenda items will be discussed. A list of action items will be noted at the bottom of the agenda, where each item will be assigned a specific team member for implementation.
+\\
+\\
+The weekly Monday meetings will be conducted in an agile/scrum-like approach. 
+\begin{itemize}
+    \item The first \textbf{15-30 minutes} will be dedicated to a standup, where each person will be giving a brief update on their progress for the prior week, their current plan/task/actions items for the upcoming week, and any roadblocks or questions they may have that concerns other parts of the system.
+    \item The remaining time meeting is allotted for additional development work, peer programming and/or tech debt. Time should be used effectively based on the upcoming deadlines for that week.
+\end{itemize}
 
-\wss{How will the meetings be structured?  There should be a chair for all meetings.  There should be an agenda for all meetings.}
+\newpage
 
 \section{Team Communication Plan}
 
-\wss{Issues on GitHub should be part of your communication plan.}
+\begin{itemize}
+    \item The main source of communication between the team will be via \textbf{Discord}. Meetings that need to be conducted virtually will be done through the dedicated voice channel (\textit{General}) within the team server.
```

Comment:
NIT: do we have to specify the server channel name?

Reply:
dont think so